tools/tests.py

```diff
@@ -187,12 +189,6 @@
         "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB],
         "automated": True,
         "peripherals": ["analog_loop"],
-        "mcu": ["LPC1768", "LPC2368", "LPC2460", "KL25Z", "K64F", "K66F", "K22F", "LPC4088", "LPC1549",
```

Comment:
Its not so much that they wouldn't have analog in or out supported, its that the pin names used for the analog loop are only defined for the targets in the `mcu` list. It would lead to a build error.